src/bridging/const.ts

```diff
@@ -3,6 +3,6 @@ import { RAW_FILES_PATH } from '../common/consts/path'
 export const RAW_PROVIDERS_FILES_PATH = `${RAW_FILES_PATH}/src/bridging/providers`
 // Based on https://dashboard.tenderly.co/shoom/project/simulator/a5e29dac-d0f2-407f-9e3d-d1b916da595b
 export const DEFAULT_GAS_COST_FOR_HOOK_ESTIMATION = 240_000
-// Based on https://dashboard.tenderly.co/shoom/project/tx/0xe95a78b313530884604aff9954dc4bd5a70bfa97950cc6678ed1450846b37fa1/gas-usage
-export const COW_SHED_PROXY_CREATION_GAS = 260_000
+// Based on https://dashboard.tenderly.co/shoom/project/tx/0x2971aee9aa7237d24b254da8ccd4345ff77410c8829c8e825e9a02cb2cece5e6/gas-usage
+export const COW_SHED_PROXY_CREATION_GAS = 360_000
```

Comment:
I don't love the hardcoding tbh, but not sure how to make it reliable and more tight.
Here, I assume it worked before and you found a case that didn't work great and that made you increase it over a 38%. This is translates to a more expensive quote and a less competitive price.

Also, I checked the TX you shared, maybe I'm checking it wrongly, but is not 160K?
<img width="1515" alt="image" src="https://github.com/user-attachments/assets/1b14bc97-a9ac-4b4a-a41b-c9badd83a7b6" />

Reply:
>Also, I checked the TX you shared, maybe I'm checking it wrongly, but is not 160K?

@anxolin are you talking about [this one](https://dashboard.tenderly.co/shoom/project/tx/0x2971aee9aa7237d24b254da8ccd4345ff77410c8829c8e825e9a02cb2cece5e6/gas-usage)?

If yes, here we see:
 - _initializeProxy 287,690 GAS
 - _initializeEns 33,664 GAS
 
 CALL - 163,015 Gas is a hook execution

<img width="995" alt="image" src="https://github.com/user-attachments/assets/007901e3-031f-4989-9bfa-a8be466cb907" />

Reply:
I also don't like hardcoding it :(
But you know that it's not trivial to estimate.
I would came back to that problem a bit later, once we finish other tasks.